### PR TITLE
azure - automation-account, automation-account.filters.variable-value

### PR DESCRIPTION
**Pull request by dmytro-afanasiev**
Use case:
```yaml
policies:
  - name: automation-acc
    resource: azure.automation-account
    filters:
      - type: variable
        attrs:
          - type: value
            key: properties.isEncrypted
            value: false
```

### CODE REVIEW

File: tools/c7n_azure/c7n_azure/resources/automation_account.py [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+        enum_spec = ('automation_account', 'list', None)
+        default_report_fields = (
+            'name',
+            'location',
+            'resourceGroup'
+        )
+        resource_type = 'Microsoft.Automation/automationAccounts'
+
+
+@AutomationAccount.filter_registry.register('variable-value')
```
Comment (kapilt) [inline tools/c7n_azure/c7n_azure/resources/automation_account.py:37]: naming, 'variable' suffices

Comment (dmytro-afanasiev) [reply tools/c7n_azure/c7n_azure/resources/automation_account.py]: Renamed

File: tools/c7n_azure/c7n_azure/resources/automation_account.py [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+
+    .. code-block:: yaml
+
+        policies:
+          - name: automation-account
+            resource: azure.automation-account
+    """
+
+    class resource_type(ArmResourceManager.resource_type):
+        doc_groups = ['Network']
```
Comment (kapilt) [inline tools/c7n_azure/c7n_azure/resources/automation_account.py:24]: its not really Network re doc group

Comment (dmytro-afanasiev) [reply tools/c7n_azure/c7n_azure/resources/automation_account.py]: Changed to `Integration`. I'm not quite sure whether it really can be interpreted as `Integration` kind of resource but other groups seem even less appropriate to me. I would suggest using `Management` group. But is there such one?
What do you think?

**Review by kapilt — APPROVED** [review]
lgtm, thanks

**Pull request merged**